File: routes/cpp-to-java/certification/artifacts/module/target-runtime/000/RouteHarness.java
public final class RouteHarness {
    public static void main(String[] args) {
        var actual0 = Migrated.both(true, true);
        var expected0 = true;
        if (actual0 != expected0) throw new AssertionError("case 0");
        System.out.println("ELMOS_OBSERVATION\t0\tbool\t" + Boolean.toString(actual0));
        var actual1 = Migrated.both(true, false);
        var expected1 = false;
        if (actual1 != expected1) throw new AssertionError("case 1");
        System.out.println("ELMOS_OBSERVATION\t1\tbool\t" + Boolean.toString(actual1));
        var actual2 = Migrated.both(false, true);
        var expected2 = false;
        if (actual2 != expected2) throw new AssertionError("case 2");
        System.out.println("ELMOS_OBSERVATION\t2\tbool\t" + Boolean.toString(actual2));
        var actual3 = Migrated.both(false, false);
        var expected3 = false;
        if (actual3 != expected3) throw new AssertionError("case 3");
        System.out.println("ELMOS_OBSERVATION\t3\tbool\t" + Boolean.toString(actual3));
    }
}

File: routes/swift-to-rust/certification/artifacts/real-repository/route_harness.rs
include!("migrated.rs");

fn main() {
    let actual_0 = difference(20, 7);
    assert!(actual_0 == 13, "case 0");
    println!("ELMOS_OBSERVATION\t0\trust-debug\t{:?}", actual_0);
    let actual_1 = difference(3, 8);
    assert!(actual_1 == 0, "case 1");
    println!("ELMOS_OBSERVATION\t1\trust-debug\t{:?}", actual_1);
    let actual_2 = difference(4, 4);
    assert!(actual_2 == 0, "case 2");
    println!("ELMOS_OBSERVATION\t2\trust-debug\t{:?}", actual_2);
}
